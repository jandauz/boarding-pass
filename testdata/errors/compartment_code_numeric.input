M1DESMARAIS/LUC       EABC123 YULFRAAC 0834 3261001A0025 100