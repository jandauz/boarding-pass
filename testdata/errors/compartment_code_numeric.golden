bcbp: Invalid data format:
  boarding pass data:
  | "M1DESMARAIS/LUC       EABC123 YULFRAAC 0834 3261001A0025 100"
  |                                                 ^ got "1"
  |
  = reason: data for "Compartment Code" must be an alpha character
